### ISSUE TITLE
Tactile Accessibility: Does Anyone Need a Haptic Glove?

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#153]
### Links
- Paper: https://dl.acm.org/citation.cfm?id=2982175

### Abstract
- Graphical user interfaces (GUIs) are widely used on smartphones, tablets, and laptops. While GUIs are convenient for sighted users, their accessibility for blind people, who use screen readers to interact with GUIs, remains to be problematic. Even the most screen-reader accessible GUIs are far less usable for blind people compared to sighted people, because the former group cannot benefit from the geometric layout of GUIs. As a result, blind people often have to listen through a lot of irrelevant content before they find what they are looking for. Haptic interfaces (those providing tactile feedback) have the potential to make GUI interfaces more accessible and usable for blind people. Alas, mainstream computer devices do not have haptic screens that would enable high-resolution tactile feedback, and specialized haptic devices are very limited and/or are exuberantly expensive and bulky.
In this paper, we describe a low-cost haptic-glove system, FeelX, which can potentially enable usable tactile interaction with GUIs. The vision of FeelX is to enable blind users to connect it to any computer or smartphone, and then interact with it by moving their hands on any flat surface such as the desk or table. To establish the practicality and the desirability of using haptic gloves, we evaluated the initial prototype of the glove in a user study with 20 blind participants. Throughout the study, we performed a comparative evaluation of several design options for the tactile interface. The participants were asked to identify simple geometric figures such as lines, rectangles, circles, and triangles that are the basic building blocks of any GUI interface. Although the FeelX prototype is far from being a usable product, the results of the study indicate that blind users want to use haptic gloves.

- スマホやPCといったGUI操作が求められるシステムを視覚障害者が操作する際，スクリーンリーダーのような読み上げシステムを使うのは大変なので，彼らでも簡単に操作することができるような安価な手袋型のデバイスを提案

<img width="410" alt="スクリーンショット 2019-11-02 21 02 17" src="https://user-images.githubusercontent.com/26996041/68070650-5136d780-fdb4-11e9-90e0-50c97cfa9e5c.png">
<img width="441" alt="スクリーンショット 2019-11-02 21 02 22" src="https://user-images.githubusercontent.com/26996041/68070651-5136d780-fdb4-11e9-861d-8bbe9e5c9990.png">

- 長方形や円や三角形といったGUIの構成要素となる図形の判別などを行えるか20人の視覚障害者を対象に調査
